afs/afs-core/src/test/java/com/powsybl/afs/ProjectTest.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.afs;
+
+import com.powsybl.afs.mapdb.storage.MapDbAppStorage;
+import com.powsybl.afs.storage.AppStorage;
+import com.powsybl.afs.storage.DefaultListenableAppStorage;
+import com.powsybl.afs.storage.NodeGenericMetadata;
+import com.powsybl.afs.storage.NodeInfo;
+import com.powsybl.computation.ComputationManager;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+
+/**
+ * @author Amira KAHYA <amira.kahya at rte-france.com>
+ */
+public class ProjectTest {
+    private AppStorage storage;
+    private AppFileSystem afs;
+    static final String FOLDER_PSEUDO_CLASS = "folder";
```

Comment:
This constant should be declared private? Constant should be declared before instance member.
```
    private static final String FOLDER_PSEUDO_CLASS = "folder";

    private AppStorage storage;
    private AppFileSystem afs;
```